charts/topolvm/templates/storageclass.yaml

```diff
@@ -0,0 +1,21 @@
+{{ if .Values.storageclass.create }}
```

Comment:
maybe consider wrapping this in a {{range $key, $valule := .Values.storageClasses }}  and make it a hash with the name being the key? Then multiple storage classes could be defined with the chart